src/PKSim.Core/Model/Individual.cs

```diff
@@ -176,5 +180,26 @@ public override void UpdatePropertiesFrom(IUpdatable sourceObject, ICloneManager
          OriginData = individual.OriginData.Clone();
          Seed = individual.Seed;
       }
+
+      public ICache<string,IParameter> AllExpressionParametersFor(IndividualMolecule molecule)
```

Comment:
This one is a bit ugly. But I need to reconstruct our previous list of relative expression parameters